src/components/account/details/UpdateToggle.js

```diff
@@ -1,13 +1,44 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { toggleEditMode } from '../../../actions/account_details'
+import Button from './Button'
 
 import '../../../styles/account/details/update_toggle.css'
 
-export default class UpdateToggle extends Component {
+class UpdateToggle extends Component {
+  handleClick() {
+    this.props.toggleEditMode()
+  }
+
   render() {
-    return (
-      <div className="edit-details__button">
-        <span>{ this.props.name }</span>
-      </div>
-    )
+    if ( this.props.editing ) {
+      return (
+        <div className="edit-details__button--active">
+          <Button styleName="button--save">Save changes</Button>
+          <Button styleName="button--cancel"
```

Comment:
Is this over 80 characters? I'd like it on one line if it's not; it reads weird this way.